mobile/src/main/java/de/duenndns/ssl/MemorizingTrustManager.java

```diff
@@ -59,363 +55,350 @@
 /**
  * A X509 trust manager implementation which asks the user about invalid
  * certificates and memorizes their decision.
- * <p>
+ * <p/>
  * The certificate validity is checked using the system default X509
  * TrustManager, creating a query Dialog if the check fails.
- * <p>
+ * <p/>
  * <b>WARNING:</b> This only works if a dedicated thread is used for
  * opening sockets!
  */
 public class MemorizingTrustManager implements X509TrustManager {
```

Comment:
Sorry for all the changes here. This file was poorly formatted and Android Studio decided to fix it. I'll comment below on the only real change to this file.